### ISSUE TITLE
Make wrench reftests use all CPU cores.

### ISSUE THREAD

**Issue by glennw** [servo/webrender#1944]
Running the reftests is slow enough now that it's worth considering running them in parallel.

This would probably involve creating an OSMesa context per CPU core, and distributing the tests on worker threads, or something similar to that.

**Comment by jrmuizel** [comment]
It'd also be interesting to compare to running them with LLVMpipe

**Comment by glennw** [comment]
True, we should revisit that and see if we can switch to llvmpipe on wrench + servo.

**Comment by strunzoz** [comment]
has it been visited
